src/redux/features/auth/authSlise.ts

```diff
@@ -105,14 +182,28 @@ const authSlice = createSlice({
       .addCase(signupUser.rejected, (state, action) => {
         state.signupError = action.payload || "";
         state.isSignupLoading = false;
-      });
+      })
+      .addCase(refreshToken.fulfilled, (state, action) => {
+        state.accessToken = action.payload.accessToken || "";
```

Comment:
probably it will be better `action.payload.accessToken || null`

Reply:
done